sys/vane/ames.hoon

```diff
@@ -1451,6 +1451,16 @@
     ?:  ?=(%crud -.kyz)
       [[[hen [%slip %d %flog kyz]] ~] +>]
     ?:  ?=(%west -.kyz)
+      ::  a .cha path of /a/give/* is immediately acknowledged and
+      ::  routed directly out to unix
+      ::
+      ?:  ?=([%give *] q.kyz)
+        ~|  [%ames-bad-duct %give duct=gad.fox p.kyz]
```

Comment:
This will always print out `duct=~` on failure. Not sure if that's a problem, but might be worth cleaning up.

Reply:
Thanks, it should probably just print the sender and route-path.